### DISCUSSION TITLE
makedb process issue - Error: staxids output field requires setting the --taxonmap parameter

### DISCUSSION THREAD

**Discussion by Fabineves7** [bbuchfink/diamond#601] [Q&A]
I need your help with this issue. I tried to build the database "**diamond makedb --in viral_proteins_merged.faa.gz --db file --taxonmap prot.accession2taxid.FULL.gz --taxonnodes nodes.dmp --taxonnames  names.dmp**  and it looks that it is concluded successfully.

However when I run the script diamond **diamond blastx -q contigs.fa -d file.dmnd -o file.txt -k 1 --very-sensitive -f 6 qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore staxids sscinames sphylums**
it instantaneously stops with the error: 
**#CPU threads: 160
Scoring parameters: (Matrix=BLOSUM62 Lambda=0.267 K=0.041 Penalties=11/1)
Error: staxids output field requires setting the --taxonmap parameter.**

I already tried to perform the run with files zipped and unzipped and also erase all the files and made new downloads from the reference databases, but I always get the same error..

Can anyone help me with this?

**Comment by bbuchfink** [comment]
It looks like you are using an older version where the `--taxonmap` parameter was used for the alignment workflow. I recommend updating to the latest version.
